miqa/core/tasks.py

```diff
@@ -49,6 +49,21 @@ def _download_from_s3(path: str, public: bool) -> bytes:
     return buf.getvalue()
 
 
+@shared_task
+def reset_demo():
+    if not settings.DEMO_MODE:
+        raise Exception('Demo Mode is false; noop.')
+    try:
+        demo_project = Project.objects.get(name='Demo Project')
+    except Project.DoesNotExist:
+        raise Exception('No Demo Project; noop.')
```

Comment:
```suggestion
    except Project.DoesNotExist as e:
        raise Exception('No Demo Project; noop.') from e
```

If you want to keep this, [exception chaining](https://docs.python.org/3/tutorial/errors.html#exception-chaining) is a nice way to preserve the original exception too (Sentry will show both).

Reply:
Removed in [418868c](https://github.com/OpenImaging/miqa/pull/635/commits/418868c2527eca6e3bbd4f287477a48908834cce)